examples/snmp/cisco_reboot.js

```diff
@@ -0,0 +1,74 @@
+/**
+ * This Driver Collects System uptime for a cisco device and stores it in a custom driver variable.
+ * A driver action function is exposed by this driver that allows the reboot of a cisco device with "snmp-server system-shutdown" configuration set
+ * Communication protocol is SNMP
+ * Creates a single custom driver variable
+ */
+
+/**
+* @remote_procedure
+* @label Validate Association
+* @documentation This procedure verifies if the device has the needed Cisco Enterprise OID for the custom_1 action (reboot)
+*/
+function validate(){
+    var tsMsgSendOid = '1.3.6.1.4.1.9.2.9.9.0'; // Integer
```

Comment:
The variable 'tsMsgSendOid' is unclear to me, both in name meaning and in-code use. Is there a better name or explanatory comment that can be put here?